src/mavo.js

```diff
@@ -161,7 +161,9 @@ var _ = self.Mavo = $.Class({
 
 		this.observe({deep: false, attribute: true}, ({attribute}) => {
 			if (attribute.indexOf("mv-") === 0) {
-				let role = attribute?.replace(/^mv-/, "");
+				// We want to observe changes both in a backend (the mv-role attribute)
+				// and its metadata (provided via the mv-role-* family of attributes)
+				let [role, ...rest] = attribute?.replace(/^mv-/, "")?.split("-");
```

Comment:
It seems a bit misleading to use a variable that is never used. What about:

```js
let role = attribute?.replace(/^mv-/, "")?.split("-")?.[0];
```

Reply:
You are absolutely right! That's better. Fixed. Thank you. 🙏🏻